playground/app.js

```diff
@@ -176,6 +211,7 @@ class App extends Component {
               formData={this.state.formData}
               onChange={data => this.setState({formData: data.formData})}
               onSubmit={data => this.setState({formData: data.formData})}
+              fields={{geo: GeoPosition}}
```

Comment:
So that's where the connection is made. The documentation should reflect that to be easier to grasp.